pkg/checker/colours.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
```

Comment:
we probably ought to create an 'ansi' package, as this could be more useful than just in here.